src/ext/java/org/opentripplanner/ext/transmodelapi/model/plan/TripPatternType.java

```diff
@@ -98,7 +98,7 @@ public static GraphQLObjectType create(
           .name("duration")
           .description("Duration of the trip, in seconds.")
           .type(ExtendedScalars.GraphQLLong)
-          .dataFetcher(env -> itinerary(env).getDurationSeconds())
+          .dataFetcher(env -> itinerary(env).getDuration())
```

Comment:
Needs output in `long`. Same for other other getters in the transmodel API

Reply:
Done, but does the Transmodel API not do any type checks?

Reply:
No, unfortunately the `dataFetcher` builder does not know what the return type should be.